objcgen/embedder.cs

```diff
@@ -678,6 +692,8 @@ public int Compile ()
 							mtouch.Append ($"--assembly-build-target=@all=framework={LibraryName}.framework ");
 							mtouch.Append ($"--target-framework {GetTargetFramework ()} ");
 							mtouch.Append ($"\"--gcc_flags=-force_load {Path.GetFullPath (sdk_output_file)}\" ");
+							if (NativeException)
+								mtouch.Append("--marshal-managed-exceptions=throwobjectivecexception ");
```

Comment:
This should be done for mmp as well.

Also style: space before opening parenthesis in method calls.